plugins/kotlin/j2k/shared/src/org/jetbrains/kotlin/nj2k/printing/JKCodeBuilder.kt

```diff
@@ -74,39 +73,76 @@ class JKCodeBuilder(context: NewJ2kConverterContext) {
             }
         }
 
-        override fun visitTreeElementRaw(treeElement: JKElement) {
+        override fun visitTreeElement(treeElement: JKElement) {
             printer.print("/* !!! Hit visitElement for element type: ${treeElement::class} !!! */")
         }
 
-        override fun visitModifierElementRaw(modifierElement: JKModifierElement) {
+        override fun visitModifierElement(modifierElement: JKModifierElement) {
```

Comment:
Interestingly, I think calls `printLeftNonCodeElements`/`printRightNonCodeElements`  were missing for `when` and type qualifier expressions even before this refactor. I figured out some test cases to capture that behavior, but since I'm abandoning this PR I'll leave this as is.

I suspect we never actually use the `JKMethodAccessExpression` class at runtime, so we never call `visitMethodAccessExpression`. 

Also it looks like calls to `printRightNonCodeElements` and `printLeftNonCodeElements` were scattered throughout both `JKCodeBuilder` and `JKVisitorWithCommentsPrinting`, so in my (admittedly biased) opinion, it's clearer with this refactor.